mcpgrafana.go

```diff
@@ -161,7 +170,11 @@ type incidentClientKey struct{}
 
 var ExtractIncidentClientFromEnv server.StdioContextFunc = func(ctx context.Context) context.Context {
 	grafanaURL, apiKey := urlAndAPIKeyFromEnv()
+	if grafanaURL == "" {
+		grafanaURL = defaultGrafanaURL
+	}
 	incidentURL := fmt.Sprintf("%s/api/plugins/grafana-incident-app/resources/api/v1/", grafanaURL)
+	slog.Debug("Creating Incident client", "url", incidentURL, "api_key_set", apiKey != "")
```

Comment:
URL also needs to be redacted (just so all of these comments need to be resolved).